js/defaultAzureCredentialAdapter.spec.ts

```diff
@@ -0,0 +1,17 @@
+import assert from "assert";
+import { KeyVaultManagementClient } from "@azure/arm-keyvault";
+import { DefaultAzureCredentialAdapter } from "./defaultAzureCredentialAdapter";
+
+const subscriptionId = "AZUER_SUBSCRIPTION_ID";
```

Comment:
Please read these values in from the .env file that is in the root of the project.

Reply:
Use this npm https://www.npmjs.com/package/dotenv